ui/react-ui/src/components/routes/home/Home.js

```diff
@@ -6,14 +6,23 @@ let Home = () => {
   let userTasksEndpoint = '/users/{userId}/tasks';
   var userIdEndpoint = '/users/{userid}';
 
+  //test results endpoints
+  var TaskIdTestReport = 'reports/taskid';
   return (
     <Container className="text-center mt-5">
       <h2>Supported Endpoints:</h2>
       <hr/>
       <p>{usersEndpoint}</p>
       <p>{userIdEndpoint}</p>
       <p>{userTasksEndpoint}</p>
+
+      <h2>Test Results Supported Endpoints</h2>
```

Comment:
the only thing that comes to my mind is to add a margin-top to this element, it's too close to the above element.
```<h2 style={{marginTop: '6rem'}}>Test Results Supported Endpoints</h2>```

Reply:
Great. Ill resolve this. Thanks

Reply:
added some padding to resolve the issue.